themes/kelpie/assets/scss/style.scss

```diff
@@ -20,68 +20,21 @@ Normalizing styles have been helped along thanks to the fine work of
 Nicolas Gallagher and Jonathan Neal http://necolas.github.io/normalize.css/
 */
 
-@import "functions";
+@import 'functions';
 
-// Start with nice clean base styles.
-@import "base/normalize";
-@import "base/sanitize";
-@import "base/wp-normalize";
+// Start with nice, clean base styles.
+@import 'base/normalize';
+@import 'base/sanitize';
+@import 'base/wp-normalize';
 
 // Import all our variables & design tokens.
-@import "design-tokens/variables";
-@import "design-tokens/typography";
+@import 'design-tokens/**/*.scss';
```

Comment:
If you aren't cool with this we can revert it, but it seems WAY less annoying…

Reply:
OH MY GOD YOU CAN DO THIS?!?!?!

Reply:
I had to add https://github.com/octopusthink/kelpie/pull/2/files#diff-11e9f7f953edc64ba14b0cc350ae7b9dR4 but yeah. 😄